Added alternative way to store keystore passwords for macOS users (#8818)
flutter/website: src/deployment/android.md

</li>

{{site.alert.note}}

>   Note: If you'd prefer not to store your passwords in plain text
>   you can store your credentials [in the Keychain Access App](https://pilloxa.gitlab.io/posts/safer-passwords-in-gradle/).
>   In which case you can skip the first 2 lines of `key.properties`
>   file and follow the steps in the link above accordingly.

Reviewer: ```suggestion
  If you'd prefer not to store your passwords in plain text,
  you can store your credentials in the [Keychain Access App][].
  In this case, skip the first 2 lines of `key.properties`
  file and follow the steps in the Pilloxa link.
```
